src/entities/Votes/types.ts

```diff
@@ -29,3 +29,5 @@ export type VotedProposal = SnapshotVote & {
 }
 
 export type ChoiceColor = 'approve' | 'reject' | number
+
+export type SelectedVoteChoice = { choice?: string; choiceIndex?: number }
```

Comment:
Why choice and choiceIndex are optional? 🤔

Reply:
It's to be able to have an empty choice selection when you haven't voted

Reply:
```typescript
const [proposalPageState, updatePageState] = usePatchState<ProposalPageState>({
    changingVote: false,
    confirmSubscription: false,
    ...
    selectedChoice: EMPTY_VOTE_CHOICE_SELECTION,
  })
  ```